java/java.source.base/src/org/netbeans/modules/java/source/pretty/VeryPretty.java

```diff
@@ -2223,6 +2223,7 @@ private boolean isLast(JCTree tree, List<? extends JCTree> list) {
      * <li>@deprecated (see How and When To Deprecate APIs)</li>
      * </ul>
      */
+    @Deprecated
```

Comment:
This is wrong. The `@deprecated` string in the javadoc is part of a list of tags.